src/fromJSON.ts

```diff
@@ -63,11 +63,11 @@ ${xyDataCreator(data, { info: { xFactor, yFactor }, xyEncoding }).join('\n')}
     if (xFactor === undefined) xFactor = 1;
     if (yFactor === undefined) yFactor = 1;
     if (xFactor !== 1) {
-      // @ts-expect-error
+      //@ts-expect-error xFactor is always defined
```

Comment:
Just published a new version of spectra-processing

xDivide(data.x, xFactor, {output: data.x})

Should yield to the same result. @jobo could you try to replace it ? I expect map to be slower.

Same is true for yFactor